torch/functional.py

```diff
@@ -160,7 +160,7 @@ def btriunpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True):
         P = torch.eye(sz).type_as(LU_data).unsqueeze(0).repeat(nBatch, 1, 1)
         for i in range(nBatch):
             for j in range(sz):
-                k = LU_pivots[i, j] - 1
+                k = int(LU_pivots[i, j] - 1)
```

Comment:
why not LU_pivots[i, j].item() ?  That would be equivalent to before, right?

Reply:
`LU_pivots[i, j]` is always an integer so `LU_pivots[i, j].item()` and `int(LU_pivots[i, j])` are identical. I've been writing `int(...)` or `float(...)` out of habit because it's backwards compatible with 0.3. This doesn't matter much here, but it's important for the changes to examples.